migrator.go

```diff
@@ -424,6 +439,8 @@ func (m Migrator) ColumnTypes(value interface{}) (columnTypes []gorm.ColumnType,
 
 			if column.DefaultValueValue.Valid {
 				column.DefaultValueValue.String = regexp.MustCompile(`'(.*)'::[\w]+$`).ReplaceAllString(column.DefaultValueValue.String, "$1")
+				// cockroachdb, removing :::type
+				column.DefaultValueValue.String = regexp.MustCompile(`(.*):::[\w]+$`).ReplaceAllString(column.DefaultValueValue.String, "$1")
```

Comment:
The following regexp can be merged with the above one?

Reply:
Changed it to `'?(.*)\b'?:+[\w\s]+$`, this should allow an optional QUOTE and matching any number of `:` between the field name and the field type.
Testing against the strings below will always get the word `field`:
```
'field'::character varying
'field':::character varying
field::character varying
field:::character varying
```